contrib/packs/actions/workflows/create.yaml

```diff
@@ -0,0 +1,51 @@
+---
+vars:
+  pack_path: '/opt/stackstorm/packs/{{ name }}'
+  pack_group: stanley
+  pack_permissions: '775'
+
+chain:
+  -
+    name: "create directory"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "mkdir -p {{ pack_path }}"
+    on-success: "change permissions"
+  -
+    name: "change permissions"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "chmod -R {{ pack_permissions }} {{ pack_path }}"
+    on-success: "change ownership"
+  -
+    name: "change ownership"
+    ref: "core.local_sudo"
+    parameters:
+      cmd: "chgrp -R {{ pack_group }} {{ pack_path }}"
+    on-success: "render pack.yml"
+  -
+    name: "render pack.yml"
```

Comment:
Yeah, knock yourself out. Personally, I'm using them pretty inconsistently throughout my code so I'd rather we support both... not sure it worth the effort though. In this particular case, I'm ok with `.yaml`.